blog/_posts/2018-07-19-team-sunshine.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Team ☀️ 'Sunshine' on RGSoC 2018!
+layout: post
+created_at: Thu Jul 19 2018
+permalink: blog/2018-07-19-team-sunshine.md
+current: blog
+author: Sujin Lee
+twitter: sujinleeme
+categories:
+- student-posts
+---
+
+# Who we are
+<img src="/img/blog/2018/2018-07-19-team-sunshine.jpg" alt="team sunshine photo"/>
```

Comment:
It's preferable to use markdown style to add images [whenever you can] and also use image-credits class to add credits for the images. The comment applies to all the images in the file.

Reply:
Solved: 5d7288c